apps/prairielearn/elements/pl-sketch/pl-sketch.mustache

```diff
@@ -0,0 +1,43 @@
+<div class="pl-sketch" id="pl-sketch-{{uuid}}">
+    <div class="pl-sketch-controls" style="width: {{width}}px">
+        <div class="pl-sketch-control-group">
+            <div class="pl-sketch-control" id="undo-{{uuid}}"><div class="fa fa-undo fa-2x"></div></div>
+            <div class="pl-sketch-control" id="redo-{{uuid}}"><div class="fa fa-redo fa-2x"></div></div>
+            <div class="pl-sketch-control" id="pen-{{uuid}}"><div class="fa fa-pen fa-2x"></div></div>
+            <div class="pl-sketch-control" id="eraser-{{uuid}}"><div class="fa fa-eraser fa-2x"></div></div>
```

Comment:
Personally, I'm not a huge fan of having a ton of elements with uuid-based IDs. One option you can use here is to have classes, with something like:
```suggestion
            <div class="pl-sketch-control pl-sketch-undo"><div class="fa fa-undo fa-2x"></div></div>
            <div class="pl-sketch-control pl-sketch-redo"><div class="fa fa-redo fa-2x"></div></div>
            <div class="pl-sketch-control pl-sketch-pen"><div class="fa fa-pen fa-2x"></div></div>
            <div class="pl-sketch-control pl-sketch-eraser"><div class="fa fa-eraser fa-2x"></div></div>
```
This should allow you to use `.pl-sketch-pen`, for example, to refer to any pen element, or `#pl-sketch-{{uuid}} .pl-sketch-pen` to refer to a specific pen control.